extensions/standard-processors/processors/TailFile.cpp

```diff
@@ -609,6 +640,10 @@ std::vector<TailState> TailFile::findRotatedFiles(const TailState &state) const
 
   utils::file::FileUtils::list_dir(state.path_, collect_matching_files, logger_, false);
 
+  return findRotatedFiles(state, matched_files_with_mtime);
+}
+
+std::vector<TailState> TailFile::findRotatedFiles(const TailState &state, std::vector<TailStateWithMtime>& matched_files_with_mtime) const {
```

Comment:
this might be a misleading name, a `sortAndSkipMainFilePrefix` or some similar contraption could better describe its function

Reply:
Updated in d9c6648dc60a73993e7f640b50220dc26a05d138